services/ui-src/.env

```diff
@@ -1,11 +1,12 @@
 LOCAL_LOGIN=true
 API_REGION=us-east-1
 API_URL=https://c1ng7kke96.execute-api.us-east-1.amazonaws.com/dev
+API_GRAPHQL_REGION=us-east-1
+API_GRAPHQL_URL=place_holder
```

Comment:
?

Reply:
This is a placeholder for the graphql environment values. These values will be populated once Api url is generated.